src/coreclr/nativeaot/Runtime/RhConfig.cpp

```diff
@@ -3,45 +3,30 @@
 #include "common.h"
 #ifndef DACCESS_COMPILE
 #include "CommonTypes.h"
-#include "daccess.h"
 #include "CommonMacros.h"
 #include "PalRedhawkCommon.h"
 #include "PalRedhawk.h"
-#include "rhassert.h"
-#include "slist.h"
-#include "gcrhinterface.h"
-#include "varint.h"
-#include "regdisplay.h"
-#include "StackFrameIterator.h"
-#include "thread.h"
 #include "holder.h"
-#include "Crst.h"
-#include "event.h"
-#include "threadstore.h"
-#include "RuntimeInstance.h"
-#include "shash.h"
 #include "RhConfig.h"
 
 #include <string.h>
 
-bool RhConfig::ReadConfigValue(_In_z_ const TCHAR *wszName, uint64_t* pValue, bool decimal)
+bool RhConfig::ReadConfigValue(_In_z_ const char *name, uint64_t* pValue, bool decimal)
 {
-    TCHAR wszBuffer[CONFIG_VAL_MAXLEN + 1]; // hex digits plus a nul terminator.
-    const uint32_t cchBuffer = sizeof(wszBuffer) / sizeof(wszBuffer[0]);
+    char buffer[CONFIG_VAL_MAXLEN + 1]; // hex digits plus a nul terminator.
+    const uint32_t cchBuffer = ARRAY_SIZE(buffer);
 
     uint32_t cchResult = 0;
 
-#ifdef FEATURE_ENVIRONMENT_VARIABLE_CONFIG
-    TCHAR wszVariableName[64] = _T("DOTNET_");
-    assert(_tcslen(wszVariableName) + _tcslen(wszName) < sizeof(wszVariableName) / sizeof(wszVariableName[0]));
```

Comment:
If you end up deleting this, you can also delete `#define _tcslen` and friends from PalRedhawk.h.